docs/Team.md

```diff
@@ -6,3 +6,114 @@ title: "Tallwave Guidelines — Team"
+
+**Day to Day Examples**
+
+A Senior Developer can run Sprint planning and grooming meetings.
+
+They can also run a Storycarding or Feature Diagramming session, and write Technical Requirements Docs.
+
+They’ll be coding a lot, and expected to have well factored results. This code should have intelligently written tests to go along with it.
+
+---
+
+### Lead Software Developer
+
+**Traits**
+* Can lead a team of developers. This is different from management.
+* Can pick up a new technology and be productive fairly quickly
+* Does a lot of non-coding stuff such as:
+  * Requirements gathering
+  * Customer management
+  * Sales support
+* Manages clients, coaches them, and gets them on board with the Goal for the engagement.
+* Makes their practice better.
+* Demonstrates excellence in:
+  * Team dynamics
+  * Planning and estimating
+  * Keeping projects on the rails
+
+**Day to Day Examples**
+
+A Lead Dev will have a high degree of ownership in their stack. They’ll be able to help write out plans for how work should be done and set goals for a group to work towards.
+
+Leads mentor more inexperienced developers more often. This could be answering questions, pair programming, or running code reviews. Leads help Software Developers grow to Seniors and Seniors to Leads.
+
+They have more conversations with clients. Per the description, Leads will be presenting project plans to clients, getting their buy-in, and working with the PM team to keep them apprised every step of the way.
+
+## Beyond Lead
+This hasn’t been figured out yet. One answer beyond “Lead” is “Management”; however just because someone is a great developer doesn’t mean they’ll be a good manager (or even a mediocre one), so it is important to have some sort of advancement plan that does not involve transitioning to management.
+
+We can say that whatever role this is will be much more highly consultative. The knowledge that this individual has is incredibly useful and will be of great importance to our clients. They'll be able to tackle problems like "design a resilient micro-service architecture on AWS that can handle a few million sessions per month and can scale way beyond that." without breaking a sweat.
+
+## Advancement
+You've heard it said that you should dress for the job you want, well similarly, if you want to be a Senior or a Lead developer, you should already be demonstrating those traits. In other words if you ask for a promotion, it shouldn't cause your manager to hesitate.
+
+You should also have a plan and some goals to be working towards, even if you are not looking for a promotion. Goals are tricky. Like estimating software in general, it's quite easy to overestimate how much you can accomplish in a year's time. And in our business, client work has a tendency to take up as much free time as possible, so while special consideration will be given if it's been a particularly busy time, we need to strive to make forward progress.
```

Comment:
Suggest a separate section with some tips and methods on how we set goals and deliver.